source/Octopus.Cli/Commands/Releases/CreateReleaseCommand.cs

```diff
@@ -19,6 +22,7 @@ namespace Octopus.Cli.Commands.Releases
     public class CreateReleaseCommand : DeploymentCommandBase, ISupportFormattedOutput
     {
         readonly IReleasePlanBuilder releasePlanBuilder;
+        new readonly IOctopusCliCommandOutputProvider commandOutputProvider;
```

Comment:
If we dont do this, it uses the base base version of `commandOutputProvider`, which is `ICommandOutputProvider`, which doesn't have the TFS specific messages that this class uses.